packages/react-devtools-inline/README.md

```diff
@@ -8,7 +8,7 @@ This is a low-level package. If you're looking for the standalone DevTools app,
 
 This package exports two entry points: a frontend (to be run in the main `window`) and a backend (to be installed and run within an `iframe`<sup>1</sup>).
 
-The frontend and backend can be initialized in any order, but **the backend must not be activated until after the frontend has been initialized**. Because of this, the simplest sequence is:
+The frontend and backend can be initialized in any order, but **the backend must not be activated until the frontend has been initialized**. Because of this, the simplest sequence is:
```

Comment:
```suggestion
The frontend and backend can be initialized in any order, but **the backend must not be activated until the frontend initialization has completed**. Because of this, the simplest sequence is:
```